api/v1/verticadb_webhook.go

```diff
@@ -380,10 +381,23 @@ func (v *VerticaDB) validateKsafety(allErrs field.ErrorList) field.ErrorList {
 
 func (v *VerticaDB) getClusterSize() int {
 	sizeSum := 0
-	for i := range v.Spec.Subclusters {
-		sc := &v.Spec.Subclusters[i]
-		sizeSum += int(sc.Size)
+
+	if v.Annotations[vmeta.RelaxKSafetyCheckAnnotation] == strconv.FormatBool(true) {
```

Comment:
I know this is still a drag but this is not how you should deal with annotations. See how it is done for other annotations in annotations.go